xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -1830,9 +1830,10 @@ void CVideoPlayer::HandlePlaySpeed()
   {
     if (m_playSpeed == DVD_PLAYSPEED_NORMAL && !isInMenu)
     {
-      // take action is audio or video stream is stalled
+      // take action if audio stream is stalled or video stream is stalled (other than a stillframe condition)
       if (((m_VideoPlayerAudio->IsStalled() && m_CurrentAudio.inited) ||
-           (m_VideoPlayerVideo->IsStalled() && m_CurrentVideo.inited)) &&
+           (m_VideoPlayerVideo->IsStalled() && !m_VideoPlayerVideo->IsStillFrame() &&
```

Comment:
this is wrong because it won't be able to detect a stall anymore. you need to ask demuxer it video stream consists of a still frame and if so tolerate stalling.

Reply:
If the still frame is kept as a separate thing (m_stillframe), then it works.  The stream can be stalled (FLUSH, OpenStream) but still not be in a known still frame state.   As above the idea was to say if you're stalled and you're _not_ in a still frame (true && true), do what you did before (audio will pause). But if you are stalled and you're in a still frame (true && false),  skip that.

Removing this and the above tracking of a still frame flag negates the entire attempt at a partial fix, there would be no changeset at all.  Which is fine if that's what you prefer.  It does solve a lot of the problem though and makes these streams much more usable.

Reply:
I don't decide anything here. I just can tell you what works and what is wrong. This change does not work and creates additional issues, see above.